website/public/js/controllers/groupsCtrl.js

```diff
@@ -16,6 +16,13 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
     }
 
     $scope.isMemberOfGroup = function(userid, group){
+
+      // If the group is a guild, just check for an intersection with the
+      // current user's guilds, rather than checking the members of the group.
+      if(group.type === 'guild') {
+        return _.detect(Groups.myGuilds(), function(g) { return g._id === group._id });
```

Comment:
My initial test made it look like it wasn't taking effect on the site, I think I just didn't wait for it to refresh though. 

I always debate about using `!!`, for readability, but it's so minified to begin with, why not?